documentation/modules/deploying/con-hostname-verification-node-ports.adoc

```diff
@@ -29,4 +30,6 @@ ssl.endpoint.identification.algorithm=
 When configuring the client directly in Java, set the configuration option to an empty string:
 
 [source,java]
-props.put("ssl.endpoint.identification.algorithm", "");
\ No newline at end of file
+props.put("ssl.endpoint.identification.algorithm", "");
+
+Alternatively, if you know the addresses of the worker nodes where the brokers will be scheduled upfront (for example because your cluster is running on bare-metal with a limited number of available worker nodes), you can use the link:{BookURLConfiguring}#property-listener-config-altnames-reference[`alternativeNames` field ^] to add additional SANs to the broker certificates manually.
```

Comment:
```suggestion
Alternatively, if you know the addresses of the worker nodes where the brokers are scheduled, you can add them as additional SANs (Subject Alternative Names) to the broker certificates manually. 
For example, this might apply if your cluster is running on a bare metal deployment with a limited number of available worker nodes. 
Use the link:{BookURLConfiguring}#property-listener-config-altnames-reference[`alternativeNames` property ^] to specify these SANS.
```